src/core/brancher.hpp

```diff
@@ -156,7 +156,7 @@ struct Brancher {
     }
   }
 
-  void activate() {
+  void activate(SHContext *context) {
```

Comment:
We could remove this as I don't use anymore but I think it's better to have for future usages. Even to just signal activate runs in context.

Reply:
I didn't add it because I was using it in a place where I don't actually have a context in my gfx branch. but I can remove it there.